lib/frizz/remote.rb

```diff
@@ -39,7 +50,9 @@ def delete(remote_file)
     attr_reader :bucket_name, :ignorance, :options
 
     def ignore?(object)
-      ignorance.ignore?(object.key)
+      # for remote files, you don't need to ignore for gzip preference,
+      # since there won't be any gzip files on s3
+      ignorance.ignore_matched_pattern?(object.key)
```

Comment:
hmm 🤔 

I can add back the `ignore?` method in the Ignorance class, and have it take a gzip_options hash parameter. The gzip_options hash would specify whether or not to ignore any files based on gzip preference, and provide a full_path if needed.

```
def ignore?(path, gzip_options)
  ignore_matched_pattern?(path) || ignore_gzip_for_gzip_version?(gzip_options)
end
```

This way we can keep the gzip and matched pattern stuff internal to `Ignorance`, and also provide the full_path when it's needed by `Ignorance`.

I'm open to any critiques.